### PR TITLE
Gtorre collins delete api endpoint

### PR DESCRIPTION
**Pull request by gtorre**
@maddalab rfr

I tested this on my box, all is well including the collins-shell and client gems.

### CODE REVIEW

File: app/collins/controllers/actions/asset/DeleteAction.scala [hunk 1]
```diff
@@ -19,43 +23,41 @@ import collins.validation.StringUtil
...
+        Right(ActionDataHolder(reason, nuke))
+    })}
   }
 
   override def execute(rd: RequestDataHolder) = Future { rd match {
-    case ActionDataHolder(options) =>
+    case ActionDataHolder(reason, nuke) =>
       val lifeCycle = new AssetLifecycle(userOption(), tattler)
-      lifeCycle.decommissionAsset(definedAsset, options) match {
+      lifeCycle.decommissionAsset(definedAsset, reason, nuke) match {
```
Comment (maddalab) [inline app/collins/controllers/actions/asset/DeleteAction.scala]: I don't see nuke being used anywhere in decomission Asset

**Comment by gtorre** [comment]
@maddalab can I get a :+1: ?

**Comment by Primer42** [comment]
LGTM

**Comment by Primer42** [comment]
Perhaps consider rebasing and squashing your commit history before merging though.

**Comment by gtorre** [comment]
Deleting in favor of https://github.com/tumblr/collins/pull/349

**Pull request closed without merging**